activefires_pp/tests/test_spatiotemporal_alarm_filtering.py

```diff
@@ -400,3 +432,26 @@ def test_alarm_filter_runner_call_spatio_temporal_alarm_filtering_no_alarms(crea
     dummy_msg = None
     result = alarm_runner.spatio_temporal_alarm_filtering(dummy_msg)
     assert result is None
+
+
+def test_send_alarm_post_ok(fake_past_detections_dir):
+    """Test send alarm."""
+
+    alarm = json.loads(PAST_ALARMS_MONSTERAS3)
+    restapi_url = "https://httpbin.org/post"
+
+    retv = post_alarm(alarm, restapi_url)
+    assert retv is True
+
+
+def test_send_alarm_post_log_messages(caplog, fake_past_detections_dir):
+    """Test send alarm."""
+
+    alarm = json.loads(PAST_ALARMS_MONSTERAS3)
+    restapi_url = "https://httpbin.org/post"
```

Comment:
Same here, what happens if httpbin.org is down?

Reply:
FIxed, thx!